src/event-sources/aws/alb.js

```diff
@@ -2,16 +2,24 @@ const { getRequestValuesFromEvent, getMultiValueHeaders } = require('../utils')
 
 const getRequestValuesFromAlbEvent = ({ event }) => getRequestValuesFromEvent({ event })
 const getResponseToAlb = ({
+  event,
   statusCode,
   body,
-  headers,
+  headers: responseHeaders,
   isBase64Encoded
 }) => {
-  const multiValueHeaders = getMultiValueHeaders({ headers })
+  const multiValueHeaders = !event.headers ? getMultiValueHeaders({ headers: responseHeaders }) : undefined
+  const headers = event.headers
+    ? Object.entries(responseHeaders).reduce((acc, [k, v]) => {
+      acc[k] = Array.isArray(v) ? v[0] : v
+      return acc
+    }, {})
+    : undefined
```

Comment:
Could you add some comments on what we're trying to do here? Why are we using `event.headers` (request headers) for response headers?